### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by OmarMWarraich**
I have added the linter config files for html and css.
I have added the form and styled it accordingly.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,10 +15,11 @@
         <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
         <p id="description" class="description text-center">
           Thank you for taking the time to help us improve the platform
+
```
Comment (OmarMWarraich) [inline index.html:18]: Kindly refrain from using an unnecessary blank line.

File: index.html [hunk 2]
```diff
@@ -29,7 +30,8 @@ <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
             required
           />
         </div>
-        <div class="form-group">
+        <div class="Form-group">
```
Comment (OmarMWarraich) [inline index.html:33]: Kindly use the best practice for using a lower case letter for naming a class.

File: index.html [hunk 3]
```diff
@@ -29,7 +30,8 @@ <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
...
+
```
Comment (OmarMWarraich) [inline index.html:34]: Kindly do not use an unnecessary blank line.

File: style.css [hunk 4]
```diff
@@ -130,7 +130,7 @@ form {
   }
 }
 
-.form-group {
+.Form-group {
```
Comment (OmarMWarraich) [inline style.css:133]: Kindly use the changed class here accordingly.

**Review by OmarMWarraich — COMMENTED** [review]
Dear Omar, you have done a fabulous job. However, you still need to remove the pointed out changes before your project is approved.

File: index.html (hunk 1 above)
Comment (HAMMAS-SALEEM) [reply index.html:18]: - [ ] It is great that you have told the student very politely to make the following change. :+1: But you didn't give any reason for the requested change. Please add a reason for the requested change :+1:

File: index.html (hunk 2 above)
Comment (HAMMAS-SALEEM) [reply index.html:33]: - [ ] It is great that you have told the student very politely to make the following change. :+1: But you didn't give any reason for the requested change. Please add a reason for the requested change :+1:

File: index.html (hunk 3 above)
Comment (HAMMAS-SALEEM) [reply index.html:34]: - [ ] It is great that you have told the student very politely to make the following change. :+1: But you didn't give any reason for the requested change. Please add a reason for the requested change :+1:

File: style.css (hunk 4 above)
Comment (HAMMAS-SALEEM) [reply style.css:133]: - [ ] It is great that you have told the student very politely to make the following change. :+1: But you didn't give any reason for the requested change. Please add a reason for the requested change :+1:

File: index.html (hunk 1 above)
Comment (OmarMWarraich) [reply index.html:18]: Kindly refrain from using an unnecessary blank line as blank lines, spaces, or indentations are not used without a reason.

File: index.html (hunk 3 above)
Comment (OmarMWarraich) [reply index.html:34]: Kindly refrain from using an unnecessary blank line as blank lines, spaces, or indentations are not used without a reason.

File: index.html (hunk 2 above)
Comment (OmarMWarraich) [reply index.html:33]: Kindly adhere to using css best practices and avoid using a capital starting letter for a css class.

File: style.css (hunk 4 above)
Comment (OmarMWarraich) [reply style.css:133]: Kindly adhere to using css best practices and avoid using a capital starting letter for a css class.

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @OmarMWarraich :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Using Github Review feature
:heavy_check_mark: Using inline comments


Good job on implementing some of the changes requested by the previous review but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:
# Mandatory Change
- [ ] It is great that you have told the student very politely to make the following change. :+1: But you didn't give any reason for the requested change. Please add a reason for the requested change :+1:
# Optional suggestions
- [ ] Please use emojis as much as you can in your inline comments to make your review human-friendly :blush:


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
# Hello @OmarMWarraich  :wave:  , 

### Great review!
   - ✔Comments are polite suggestions rather than orders.
   - ✔Good explanation as to why the changes are requested.
   - ✔Used the Github review function correctly.

Your project is complete! Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: index.html (hunk 1 above)
Comment (OmarMWarraich) [reply index.html:18]: Great readability of your html makes it much easier to maintain in the future, as you'll be able to find elements quicker. Also, you'll never know who might need to look at your code later on.

File: index.html (hunk 2 above)
Comment (OmarMWarraich) [reply index.html:33]: Great readability of your css makes it much easier to maintain in the future, as you'll be able to find elements quicker. Also, you'll never know who might need to look at your code later on.

File: index.html (hunk 3 above)
Comment (OmarMWarraich) [reply index.html:34]: Great readability of your html makes it much easier to maintain in the future, as you'll be able to find elements quicker. Also, you'll never know who might need to look at your code later on.

File: style.css (hunk 4 above)
Comment (OmarMWarraich) [reply style.css:133]: Great readability of your CSS makes it much easier to maintain in the future, as you'll be able to find elements quicker. Also, you'll never know who might need to look at your code later on.

**Pull request merged**